README.md

```diff
@@ -29,7 +29,7 @@ build.cmd
 
 The `install` task
 will publish the package
-to Sublime Text's *Data* directory.
```

Comment:
I agree about splitting, but these lines are short!? :) 80 is a common standard?